gems/pending/Scvmm/miq_hyperv_disk.rb

```diff
@@ -150,7 +150,7 @@ def bread(start_sector, number_sectors)
       encoded_data = @parser.output_to_attribute(run_correct_powershell(read_script))
       t2           = Time.now.getlocal
       buffer       = ""
-      Base64.decode64(encoded_data).split(' ').each { |c| buffer += c.to_i.chr }
+      Base64.decode64(encoded_data).split(' ').each { |c| buffer << c.to_i.chr }
```

Comment:
@jerryk55 - I'm sorry, this make absolutely no sense to me. If the output of `Base64.decode64(encoded_data)` is a string, _consisting of string representations of numbers with spaces between each number_, that means the data **before** it was encoded was a string, _consisting of string representations of numbers with spaces between each number_. Then why encode it in the first place?

It sounds to me that you're taking binary data, converting in into a string representation of the value of each byte (separated by spaces), and then encoding the string. You should just be encoding the binary data directly.